### PR TITLE
UUID BINARY 16 it is not supported from postgresql. I replaced it by …

### PR DESCRIPTION
**Pull request by marcocast**
…using a sequence

### CODE REVIEW

File: src/main/java/org/ow2/proactive/catalog/repository/entity/KeyValueLabelMetadataEntity.java [hunk 1]
```diff
@@ -63,11 +64,13 @@
...
 
-    @GeneratedValue(generator = "uuid2")
-    @GenericGenerator(name = "uuid2", strategy = "uuid2")
-    @Column(name = "ID", columnDefinition = "BINARY(16)")
     @Id
-    private UUID id;
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "METADATA_KEY_VALUE_SEQUENCE")
+    @GenericGenerator(name = "METADATA_KEY_VALUE_SEQUENCE", strategy = "org.hibernate.id.enhanced.SequenceStyleGenerator", parameters = { @Parameter(name = "sequence_name", value = "METADATA_KEY_VALUE_SEQUENCE"),
+                                                                                                                                          @Parameter(name = "initial_value", value = "1"),
+                                                                                                                                          @Parameter(name = "increment_size", value = "1") })
```
Comment (tobwiens) [inline src/main/java/org/ow2/proactive/catalog/repository/entity/KeyValueLabelMetadataEntity.java]: Did you put the spaces on purpose?

File: src/main/java/org/ow2/proactive/catalog/repository/entity/CatalogObjectRevisionEntity.java [hunk 2]
```diff
@@ -68,11 +69,13 @@
...
 
-    @GeneratedValue(generator = "uuid2")
-    @GenericGenerator(name = "uuid2", strategy = "uuid2")
-    @Column(name = "ID", columnDefinition = "BINARY(16)")
     @Id
-    private UUID id;
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "CATALOG_OBJECT_REVISION_SEQUENCE")
+    @GenericGenerator(name = "CATALOG_OBJECT_REVISION_SEQUENCE", strategy = "org.hibernate.id.enhanced.SequenceStyleGenerator", parameters = { @Parameter(name = "sequence_name", value = "CATALOG_OBJECT_REVISION_SEQUENCE"),
+                                                                                                                                               @Parameter(name = "initial_value", value = "1"),
+                                                                                                                                               @Parameter(name = "increment_size", value = "1") })
```
Comment (tobwiens) [inline src/main/java/org/ow2/proactive/catalog/repository/entity/CatalogObjectRevisionEntity.java]: Is the space between the annotations on purpose?

File: src/main/java/org/ow2/proactive/catalog/repository/entity/KeyValueLabelMetadataEntity.java (hunk 1 above)
Comment (marcocast) [reply src/main/java/org/ow2/proactive/catalog/repository/entity/KeyValueLabelMetadataEntity.java]: which spaces?

File: src/main/java/org/ow2/proactive/catalog/repository/entity/CatalogObjectRevisionEntity.java (hunk 2 above)
Comment (marcocast) [reply src/main/java/org/ow2/proactive/catalog/repository/entity/CatalogObjectRevisionEntity.java]: it is done by spotlessApply plugin

**Pull request merged**